public/pages/ShrinkIndex/container/ShrinkIndex/ShrinkIndex.test.tsx

```diff
@@ -17,6 +17,23 @@ import { CoreServicesConsumer, CoreServicesContext } from "../../../../component
 import ShrinkIndex from "./ShrinkIndex";
 import { ModalProvider, ModalRoot } from "../../../../components/Modal";
 import { BrowserServices } from "../../../../models/interfaces";
+import { getApplication, getNavigationUI, getUISettings } from "../../../../services/Services";
+
+jest.mock("../../../../services/Services", () => ({
+  ...jest.requireActual("../../../../services/Services"),
+  getUISettings: jest.fn(),
```

Comment:
OSD provides mocks for stuff like uiSettings. It would be safer to use theirs than reinvent the wheel

Reply:
Will check. Thanks!